README.md

```diff
@@ -1,2 +1,72 @@
 # go-sc2ai
-Go version of the Starcraft II AI API
+
+Go implementation of the Starcraft II AI API
+
+## Quick start
+
+### Prerequisites
+
+- Install golang >= 1.11  
+    https://golang.org/dl/
+- Install Starcraft II game  
+    https://starcraft2.com/en-us/ (free version is okay)
+- Download Ladder  
+    https://drive.google.com/file/d/18lmZEzzZEP1VhqmHKsiSn9g7P2BDutNJ/view  
```

Comment:
Might be better to just point to the channel topic in the #ladder channel in Discord so this link doesn't get out of date?

Reply:
Yeah, added info about !maps command